lib/within.js

```diff
@@ -21,18 +21,24 @@ function within(context, fn) {
       if (helpers[helper]._withinBegin) recorder.add('start within block', () => helpers[helper]._withinBegin(context));
     });
 
-    fn();
+    try{
+      fn();
+    }
+    catch(e){
+      //need suggestion how to report error with respect to codeceptJS
```

Comment:
@DavertMik travis tests are failing and I couldn't understand why. Could you please share some guidance on that ?